examples/07-periodic_hills/periodic_hills.prm

```diff
@@ -6,18 +6,18 @@
 subsection simulation control
   set method                       = bdf1
   set number mesh adapt            = 0
-  set output name                  = periodic_hills-output
+  set output name                  = periodic_hills_re5600_dt1E-1-output
```

Comment:
No need for such a complex output name. You can just call it periodic_hill_output  or just hill_output or whatever. Make it simple :)